welcome_wizard/views.py

```diff
@@ -236,50 +259,27 @@ def check_data(cls):
             ),
         ]:
             completed = nautobot_object.objects.exists()
-            try:
-                Merlin.objects.filter(name=var_name).update(completed=completed)
-                merlin_id = Merlin.objects.get(name=var_name)
-            except Merlin.DoesNotExist:
-                merlin_id = None
+            Merlin.objects.update_or_create(
+                name=var_name,
+                defaults={
+                    "completed": completed,
+                    "ignored": False,
+                    "nautobot_model": nautobot_object._meta.label,
+                    "nautobot_add_link": new_url,
+                    "merlin_link": wizard_url,
+                    "nautobot_list_link": list_url,
```

Comment:
Will you update the variable names in this loop to match the field names on the Merlin model? Trying to keep track of these is difficult.